cmd/daemon/main_test.go

```diff
@@ -1,13 +1,12 @@
 package main
 
 import (
-	"fmt"
 	"testing"
 )
 
 // TODO Delete -- used temporarily to get hack project merged
 func TestTempTest(t *testing.T) {
 	if !true {
-		fmt.Errorf("temporary test failed")
+		t.Errorf("temporary test failed")
```

Comment:
do we still need this test? I don't really understand it since it never runs (if !true), and there are other tests now

Reply:
I only updated it because a warning showed up in my editor, but nope, we don't need it anymore